manual_flagging/yaml/inputs.yaml

```diff
@@ -13,7 +13,15 @@ run_note: |
 # without flags, it assigns a version number of "1". The latest
 # version of each sale flag can be found in the "default"."vw_pin_sale" view.
 
-manual_update: true
+manual_update: false
+
+# manual_update_only_new_sales: A boolean that only is in play when
+# manual_update = true. If true, the sales val pipeline will only
+# flag sale that currently have no existing flag in the sale.flag
+# table. In other words, it will flag unseen sales, not sales that
+# the sales val model has already given a flag value to.
+
+manual_update_only_new_sales: true
```

Comment:
**[Nitpick, non-blocking]** Same here! Plus a couple typos:

```suggestion
# manual_update_only_new_sales: A boolean that only is in play when
# manual_update = true. If true, the sales val pipeline will only
# flag sales that currently have no existing flag in the sale.flag
# table. In other words, it will flag unseen sales, not sales that
# the sales val model has already given a flag value to.
manual_update_only_new_sales: true
```